systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicUtils.java

```diff
@@ -73,6 +73,12 @@ public static void waitForKafkaTopicCreationByNamePrefix(String topicNamePrefix)
         );
     }
 
+    public static void deleteKafkaTopicWithWait(String topicName) {
+        LOGGER.info("Deleting topic {} in namespace {}", topicName, kubeClient().getNamespace());
+        KafkaTopicResource.kafkaTopicClient().inNamespace(kubeClient().getNamespace()).withName(topicName).delete();
+        waitForKafkaTopicDeletion(topicName);
+    }
```

Comment:
do we still need this method?

Reply:
Not at all, if we will need it in future, we add it in different PR, thanks :)